backend/baseclass.pm

```diff
@@ -1240,7 +1241,8 @@ sub new_ssh_connection {
         }
     }
 
-    my $ssh = Net::SSH2->new;
+    # timeout requires libssh2 >= 1.2.9 so not all versions might have it
```

Comment:
So what does this mean? The following could can fail? What is the comment telling me?

Reply:
I expect that the timeout would be ignored